### PR TITLE
Fixed Auth On Frontend / Backend By Adding Bearer String To useOktaAxios Token

### PR DESCRIPTION
**Pull request by ahmedseragcodes**
# Description

overall: got incidents rendering WITH authentication & okta validation enabled

specific: added bearer string to useOktaAxios function and component, which gets authorization header working properly for backend APIs and okta validation

#Corresponding backend PR

https://github.com/Lambda-School-Labs/human-rights-first-police-be-a/pull/90

##Loom Recording 

https://www.loom.com/share/475123ca43bf44748ab3d76cfa518f8c

##Trello

https://trello.com/c/W49eewfD

##Fixes 

okta validation, authentication

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)

## Change Status

- [ ] Complete, tested, ready to review and merge, needs unit tests inspected or created

# Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

**Review by ooladuwa — APPROVED** [review]
Placed Bearer in the correct place so that it passes properly and takes care of the token.  Wasn't sure that in approvedIncidents the "x" was supposed to stay changed to "delete"

**Review by austin-carman — COMMENTED** [review]
Naj, your comment made it sound like you weren't sure if it should be "x" or "delete" here. Could you expound on the reasoning for changing it?

File: src/hooks/useOktaAxios.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ export default function useOktaAxios() {
   return axios.create({
     baseURL: process.env.REACT_APP_BACKENDURL || '',
     headers: {
-      authorization: isAuthenticated ? `${idToken.value}` : '',
+      authorization: isAuthenticated ? `Bearer ${idToken.value}` : '',
     },
```
Comment (lgarcia2689) [inline src/hooks/useOktaAxios.js:12]: Very happy to have this fixed, it was vital for the this program to function correctly. Was able to test and verify code work and fixed issue with not displaying incidents

**Review by lgarcia2689 — APPROVED** [review]
Very happy to have this fixed, it was vital for the this program to function correctly. Was able to test and verify code work and fixed issue with not displaying incidents

File: src/hooks/useOktaAxios.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ export default function useOktaAxios() {
   return axios.create({
     baseURL: process.env.REACT_APP_BACKENDURL || '',
     headers: {
-      authorization: isAuthenticated ? `${idToken.value}` : '',
+      authorization: isAuthenticated ? `Bearer ${idToken.value}` : '',
```
Comment (paulstgermain) [inline src/hooks/useOktaAxios.js:11]: A simple change, but a valuable and insightful one in light of the necessary changes shown in the [paired backend changes](https://github.com/Lambda-School-Labs/human-rights-first-police-be-a/pull/90). By including this authentication we grow closer to a working, secure application - great work!

**Review by paulstgermain — APPROVED** [review]
## Video Walkthrough Feedback

- You have a very steady, thorough style of walking through your work, swapping back and forth between code and examples at a masterful pace befitting someone trying to educate their viewer - and these are qualities you should proudly carry into your actual interviews when you begin talking about your work!
- You very clearly know exactly what your code and imported methods do, explaining every line very well without being too rambling - nice!
- Speaking of examples, I love that you showed test runs of your updated API that pairs with this PR to prove that your code is working as intended on all fronts!

## Code Review

See the attached comments!

**Pull request merged**